src/vos/README.md

```diff
@@ -699,6 +699,24 @@ Metadata totals:
 Total bytes with user data: 2079047482K
 ```
 
+It is possible to take into account the space allocated by the DAOS File System. An example input yaml is installed to /etc/vos_dfs_sample.yaml. This file can also be generated using the information of the actual DAOS installation.
```

Comment:
is the newly generated file presently the same as the one that gets installed?   If not, we should update the installed one.  Or we should remove it altogether and just ask them to generate it

Reply:
Ah, I see you added code to generate it, never mind the comment